handlers/handler_api.go

```diff
@@ -0,0 +1,93 @@
+package handlers
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+// RegisterSQSHandler registers endpoints
+func (handler *SQSHandler) RegisterSQSHandler() {
+	http.HandleFunc("/server", handler.ServiceHandler)
+	http.HandleFunc("/sqs", handler.SQSHandler)
+	http.HandleFunc("/job", handler.HandleJob)
+}
+
+// ServiceHandler handles stop/start the SQS querrying service
+func (handler *SQSHandler) ServiceHandler(w http.ResponseWriter, r *http.Request) {
+	if r.Method == "PUT" {
+		val := r.URL.Query().Get("start")
+		if val == "true" {
+			handler.StartServer()
+		} else if val == "false" {
+			handler.ShutdownServer()
+		}
+
+	} else {
+		http.Error(w, "Not supported request method.", 405)
```

Comment:
you might look into setting up the router using something like `mux`, then the methods are checked automatically and these error checks are unnecessary